### ISSUE TITLE
How to add dropout to indrnn cells

### ISSUE THREAD

**Issue by psyxusheng** [batzner/indrnn#22]
Looking forward to reply~~

**Comment by Sunnydreamrain** [comment]
Not sure how it is done in tensorflow, but in pytorch, you can write a fuction to share the dropout mask over time as in the following line.  
https://github.com/Sunnydreamrain/IndRNN_pytorch/blob/master/action_recognition/Indrnn_action_network.py#L26

Anyway, the key is to share the dropout mask over time, which should be easily done.

**Comment by psyxusheng** [comment]
Thank you
